pool.go

```diff
@@ -334,13 +349,44 @@ func (p *Pool) err(err error) {
 	}
 }
 
+func (p *Pool) traceConnectDone(conn *ioErrConn, reason string, err error) {
+	if p.opts.pt.ConnectDone != nil {
+		var connectTime time.Duration
+		if conn != nil {
+			connectTime = conn.ConnectTime
+		} else {
+			connectTime = 0
```

Comment:
This is unnecessary since `connectTime` is already 0.

Reply:
I'll fix it.